gno.land/pkg/integration/testing_node.go

```diff
@@ -4,13 +4,14 @@ import (
 	"path/filepath"
 	"time"
 
+	"golang.org/x/exp/slog"
+
```

Comment:
Could you please try running gofumpt or a similar tool to improve the consistency of goimports?

Reply:
Moved:

[1c29d7c](https://github.com/gnolang/gno/pull/1302/commits/1c29d7ca07792a32679dbb1ba838efb0ac736de8)